homeassistant/components/cast/__init__.py

```diff
@@ -8,6 +8,7 @@
 import voluptuous as vol
 
 from homeassistant.components.media_player import BrowseMedia
+from homeassistant.components.media_player.const import MediaType
```

Comment:
_Please correct me if I'm wrong._

I believe we should only import things from the top level integration. I.e. all import should be just from `homeassistant.components.media_player`. That `MediaType` is defined in `const.py` is an implementation detail. That's also why we import (and later would reexport it via `__all__`) in `__init__.py`.

I couldn't find a better reference but the discussion around that began with https://github.com/home-assistant/core/pull/63820#issuecomment-1013885233